zerocopy-derive/tests/ui/padding_check.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 The Fuchsia Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#[macro_use]
+extern crate zerocopy;
+
+fn main() {}
+
+// These tests exercise our checks for padding bytes in types.
```

Comment:
Can you also mention why they're in a separate file from the other tests?

Reply:
Originally I thought I needed them in a separate pass. Moved back into the main files.